src/components/user/user-avatar.tsx

```diff
@@ -0,0 +1,56 @@
+import React, { ComponentProps } from 'react';
+
+import './user-avatar.css';
+import { Icon } from '../icon/icon';
+import { DropDown } from '../dropdown/dropdown';
+import { cva, VariantProps } from 'class-variance-authority';
+
+const UserAvatarVariants = cva(
+    'base-use-avatar', 
+    {
+        variants: {
+            variant: {
+                solid: 'solid',
+                outline: 'outline'
+            }, 
+            mode: {
+                dark: 'dark', 
+                light: 'light'
+            }
+        }, 
+        defaultVariants: {
+            variant: 'solid',
+            mode: 'light'
+        }
+    }
+)
+
+interface UserAvatarProps extends ComponentProps<'div'>, VariantProps<typeof UserAvatarVariants>{
+    variant?: 'solid' | 'outline'
+    mode?: 'light' | 'dark'
+    photoUrl: string, 
+    userName: string,
+    onClick: () => void;
+}
+
+export const UserAvatar = ({variant, mode, ...props}: UserAvatarProps) => {
+    const [active, setActive] = React.useState(false);
+
+    return (
+        <div className={UserAvatarVariants({variant, mode})} {...props}>
```

Comment:
```suggestion
        <div className='base-user-avatar' {...props}>
```

Isso aqui seria mais pro caso de você estar usando a classe `base-user-avatar`, que você tinha setado no CVA, pra alguma coisa. Mas vi no CSS que você não tá usando essa classe, então pode tirar também se preferir.